portfolio/src/main/webapp/style.css

```
@@ -106,7 +107,7 @@
     font-size: 40px;
   }
   body {
-    background-image: url("Images/bg.jpg");
+    background-image: url("Images/layout.webp");
```

[4.2.8](https://google.github.io/styleguide/htmlcssguide.html#CSS_Quotation_Marks): Do not use quotation marks in URI values (`url()`).
```suggestion
    background-image: url(Images/layout.webp);
```